RELEASE-NOTES.txt

```diff
@@ -1,6 +1,10 @@
+14.3
+-----
+ * Block editor: New Group block for nesting blocks.
```

Comment:
We don't need to add this since we are already mentioning it in gutenberg-mobile release notes.